fdbserver/DataDistribution.actor.cpp

```diff
@@ -1875,6 +1874,73 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 		return found;
 	}
 
+	// Check if the serverTeam belongs to a machine team; If not, create the machine team
+	void checkAndCreateMachineTeam(Reference<TCTeamInfo> serverTeam) {
```

Comment:
this needs to update the serverTeam's pointer to a machine team to point to the newly created team